src/main/java/bio/terra/janitor/service/pubsub/TrackedResourceSubscriber.java

```diff
@@ -0,0 +1,65 @@
+package bio.terra.janitor.service.pubsub;
+
+import bio.terra.generated.model.CreateResourceRequestBody;
+import bio.terra.janitor.app.configuration.TrackResourcePubsubConfiguration;
+import bio.terra.janitor.common.exception.InvalidMessageException;
+import bio.terra.janitor.service.janitor.JanitorService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.cloud.pubsub.v1.AckReplyConsumer;
+import com.google.cloud.pubsub.v1.MessageReceiver;
+import com.google.cloud.pubsub.v1.Subscriber;
+import com.google.pubsub.v1.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/** Subscribes track resource message and update Janitor record. */
```

Comment:
nit: `Subscribes to tracked resource topic to update the Janitor database for new tracked resources.`